src/components/tiles/text/dialog/use-link-dialog.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from "react";
+import { CustomElement, Editor, EFormat, ReactEditor, Transforms } from "@concord-consortium/slate-editor";
+
+import { useCustomModal } from "../../../../hooks/use-custom-modal";
+
+import LinkIcon from "../../../../assets/icons/text/link-text-icon.svg";
+import './use-link-dialog.scss';
+
+interface IContentProps {
+  setUrl: React.Dispatch<React.SetStateAction<string>>;
+  text: string;
+  url: string;
+}
+export const LinkDialogContent = ({ setUrl, text, url }: IContentProps) => {
+  return (
+    <div className="link-dialog-content">
+      <p>Link text: {text}</p>
+      <input
+        placeholder="Url"
+        onChange={e => setUrl(e.target.value)}
+        type="text"
```

Comment:
Wouldn't it make sense to use `type="url"` here?

Reply:
I agree `type="url"` would make sense.

Super picky suggestion, sorry, but I also think the placeholder value should be "URL" instead of "Url".

You may also want to consider adding `spellcheck="false"` to the field. I don't personally think that's all that important, but project team members have previously asked for spellcheck to be disabled on fields that are likely to contain words you wouldn't find in a dictionary.